install/yaml/install.yaml

```diff
@@ -5767,6 +5768,127 @@ spec:
                           anyOf:
                             - type: integer
                             - type: string
+                    chain:
```

Comment:
I would not expect this to be in the `install.yaml` yet. You'll have to do your tests by enabling the feature gate - you can use the `FEATURE_GATES` variable to turn it on for `make install`: https://github.com/googleforgames/agones/blob/0f7c9436ed324bf1bf44dd92fe0eb32e3c621ddb/build/Makefile#L430

Reply:
Since this is a dev feature, does it matter if I add the alpha feature gates variable in the Makefile (for testing purposes) ?